.github/workflows/daily-linkcheck.yml

```
@@ -0,0 +1,21 @@
+name: Daily link check
+on:
+  pull_request:
+  schedule:
+    - cron: '00 19 * * *' # every day at 19:00 UTC
+
+  workflow_dispatch:
+    # Manual trigger
+
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@main
```

I think it is recommended to use a stable version to not get surprised with updates.
```suggestion
        uses: actions/checkout@v5
```